src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponentSpec.scala

```diff
@@ -2,6 +2,7 @@ package org.broadinstitute.dsde.workbench.leonardo.db
 
```

Comment:
^ why did `MockGoogleDataprocDAO.scala` sneak in with no changes?

Reply:
Seems like a rebase issue